metals/src/main/scala/scala/meta/internal/metals/DefinitionProvider.scala

```diff
@@ -133,8 +133,19 @@ final class DefinitionProvider(
           .map(reportBuilder.setFallbackResult)
       )
 
+    // Scala 3 prior to 3.7.0 has a bug where the definition is much slower
+    def scala3DefinitionBugFixed: Boolean =
```

Comment:
The fix is also in `3.3.3` and `3.3.1`, since we still publish pc for them. BTW, maybe it is about time to stop and delete Scala 3 pc from metals?

Reply:
Maybe, though I don't want to do this in the same PR. I added another condition for now